frontend/lib/ui/user_avatar.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+
+import '../models/user.dart';
+
+Color userIdToColor(int userId) {
+  const colorGroups = Colors.accents;
+  return colorGroups[userId % colorGroups.length];
+}
+
+class UserAvatar extends StatelessWidget {
+  const UserAvatar({super.key, required this.user});
+
+  final User user;
+
+  @override
+  Widget build(BuildContext context) {
+    return CircleAvatar(
+      backgroundColor: userIdToColor(user.id),
+      child: Tooltip(
+        message:
+            '${user.name}\n${user.launchpadEmail}\n${user.launchpadHandle}',
```

Comment:
I suggest ordering it like `name, handle, email` like we have it on MM (skipping the titler ofc)
![image](https://github.com/canonical/test_observer/assets/84857215/3e8f17b1-c0b6-4a1c-a14a-279b3d14e572)